lib/easy_upnp/upnp_device.rb

```diff
@@ -37,8 +37,8 @@ def host
       @host ||= URI.parse(@service_definitions.first[:location]).host
     end
 
-    def device_name
```

Comment:
I think I'd prefer to keep this method.  It can just call `description`, but would like to provide backwards compatibility.